bin/export-db-to-sqlite.sh

```diff
@@ -50,8 +50,6 @@ tables_to_wipe_after_initial_migrate=(
     "auth_group_permissions"
     "auth_group"
     "auth_permission"
-    "auth_user_groups"
-    "auth_user_user_permissions"
```

Comment:
These tables have no data set by initial migrations because they are User-related, and by default there are no users in the system, so there will be no data in these tables to link a Permission to a User or to their Group